### ISSUE TITLE
Ask for usage when the answer is correct

### ISSUE THREAD

**Issue by albsierra** [JuezLTI/codetest#59]
https://github.com/JuezLTI/codetest/blob/2682058d74aafe7449858a83860deeae97a5e115/assets/scripts/codetest.js#L615

Some questions in usage modal, such as difficulty, isn't known to the students until they right solve the exercise and not the first time they answer it.

To achieve this, we need to change [code to include modal into StudentView](https://github.com/JuezLTI/codetest/blob/2682058d74aafe7449858a83860deeae97a5e115/views/pages/student-view.php.twig#L36) to:
```
        {% if not user.isInstructor(main.getCtId()) %}
            {{ include('exercise/usageModal.php.twig') }}
        {% endif %}
```
We also need to move [the code to show the modal](https://github.com/JuezLTI/codetest/blob/2682058d74aafe7449858a83860deeae97a5e115/assets/scripts/codetest.js#L616) after the [validator returns the grade of exercise](https://github.com/JuezLTI/codetest/blob/2682058d74aafe7449858a83860deeae97a5e115/actions/AnswerExercise.php#L51).

**Comment by albsierra** [comment]
We could even add a "Cancel" button to allow students to continue with subsequent exercises without answering the usage form.

**Comment by RaZaC-WP** [comment]
I've just created a PR(https://github.com/JuezLTI/codetest/pull/65) that resolve this issue

**Comment by frasese** [comment]
Resolved in 1d223e220fb8096c2a328f7f140a2a2eaf4f115c
